src/views/dependencyExplorer.ts

```diff
@@ -29,7 +29,10 @@ export class DependencyExplorer implements Disposable {
         context.subscriptions.push(
             window.onDidChangeActiveTextEditor((textEditor: TextEditor) => {
                 if (this._dependencyViewer.visible && textEditor && textEditor.document && Settings.syncWithFolderExplorer()) {
-                    this.reveal(textEditor.document.uri);
+                    const uri: Uri = textEditor.document.uri;
+                    if (uri.scheme === "file") {
```

Comment:
add `jdt` to cover the content provider for .class files?

Reply:
make sense

Reply:
Can the `textEditor.document.uri` be empty? If opening an Untitled file, what's the uri like?

Reply:
It's `'untitled:Untitled-1'`